src/frontend/python/chiquito/cb.py

```diff
@@ -222,3 +227,93 @@ def to_constraint(v: ToConstraint) -> Constraint:
         raise TypeError(
             f"Type `{type(v)}` is not ToConstraint (one of Constraint, Expr, int, or F)."
         )
+
+
+@dataclass
+class LookupTableRegistry:
```

Comment:
@qwang98 I don't think we need a LookupTableRegistry, this was necessary in rust to have the `Copy` constraint, but in python the table is a reference.

Reply:
Removed and fixed!